resources/assets/js/components/MicrotaskingCallToAction.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <CollapsibleSection id="cta" class="px-4 pb-4 pt-4 pt-md-0 lineheight" :show-horizontal-rule="false">
+        <template slot="title">
+            {{ translatedTitle }}
+        </template>
+
+        <template slot="title-right">
+            <b-img class="d-none d-sm-block ml-auto doodle" src="/images/wire-strippers.svg" />
+        </template>
+
+        <template slot="content">
+            <div class="content pt-3">
+                <p>
+                    {{ translatedDescription }}
+                </p>
+                <div style="align-self:center; justify-self:right">
+                <a href="/mobifix" style="align-self: center" class="btn btn-primary pull-right">{{ translatedGetInvolved }}</a>
+                </div>
+            </div>
+
+        </template>
+    </CollapsibleSection>
+</template>
+
+<script>
+import CollapsibleSection from './CollapsibleSection'
+
+export default {
+  components: {CollapsibleSection},
+  computed: {
+    translatedTitle() {
+      return this.$lang.get('microtasking.cta.title')
+    },
+    translatedDescription() {
+      return this.$lang.get('microtasking.cta.description')
+    },
+    translatedGetInvolved() {
+      return this.$lang.get('microtasking.cta.get_involved')
+    },
+  }
+}
+</script>
+
+<style scoped lang="scss">
+@import 'resources/global/css/_variables';
+@import '~bootstrap/scss/functions';
+@import '~bootstrap/scss/variables';
+@import '~bootstrap/scss/mixins/_breakpoints';
+
+#cta {
+    background-color: #4aaebc;
```

Comment:
I think you can use $brand-light for this colour.

Reply:
Ah yes thanks - will use the variable.